packages/react-components/src/index.ts

```diff
@@ -1,6 +1,7 @@
 import './index.scss';
 
 export { DesignTokens } from './themes/designTokens';
```

Comment:
We could fix this also
```suggestion
export { DesignToken } from './themes/design-oken';
```

Reply:
👍 But we need to remember to include this as (small) breaking change in release docs, as we use `DesignTokens` enum in another app.